File: java/src/com/company/BinaryTree.java
package com.company;

import java.util.HashMap;

public class BinaryTree {
    private String root;
    private HashMap<String, Node> nodes;

    public BinaryTree() {
        this.root = null;
        this.nodes = new HashMap<>();
    }

    public String getRoot() {
        return root;
    }

    public void setRoot(String root) {
        this.root = root;
    }

    public HashMap<String, Node> getNodes() {
        return nodes;
    }

    public void setNodes(HashMap<String, Node> nodes) {
        this.nodes = nodes;
    }

    @Override
    public String toString() {
        return "BinaryTree {" +
               "root:'" + root + '\'' +
               ", nodes:" + nodes +
               '}';
    }
}

File: php/BinaryTree.php
<?php


class BinaryTree
{
    private $root;
    private $nodes;

    public function __construct()
    {
        $this->root = null;
        $this->nodes = array();
    }

    /**
     * @return array
     */
    public function getNodes()
    {
        return $this->nodes;
    }

    /**
     * @return string
     */
    public function getRoot()
    {
        return $this->root;
    }

    /**
     * @param array $nodes
     */
    public function setNodes($nodes)
    {
        $this->nodes = $nodes;
    }

    /**
     * @param string $root
     */
    public function setRoot($root)
    {
        $this->root = $root;
    }
}